docs/3.api/5.kit/5.components.md

```diff
@@ -158,6 +158,16 @@ An object with the following properties:
 
   Transpile specified path using build.transpile. If set to `'auto'`, it will set `transpile: true` if `node_modules/` is in path.
 
+#### `opt`
+
+**Required**: `false`
+
+- `prepend` (optional)
+
+  **Type**: `boolean`
+
+  If set to `true`, it will be `unshift` into Nuxt Component Dirs instand of `push`.
```

Comment:
borrowing wording from elsewhere in the docs:

```suggestion
  If set to `true`, the directory will be prepended to the array with `unshift()` instead of `push()`.
```